x/evm/types/key.go

```diff
@@ -30,6 +30,7 @@ const (
 	prefixLogs
 	prefixCode
 	prefixStorage
+	prefixTxLogNum
```

Comment:
how about
prefixTxLogNum , it's a little unclear whether it's ID or count.

prefixTxLogCount ?

Reply:
sure